gensim/sklearn_api/lsimodel.py

```diff
@@ -20,14 +15,36 @@
 
 
 class LsiTransformer(TransformerMixin, BaseEstimator):
-    """
-    Base LSI module
+    """Base LSI module.
+
+    Scikit learn interface for `gensim.models.lsimodel` for easy use of gensim with scikit-learn.
```

Comment:
please use links 
```
:class:`~gensim.model.lsimodel.LsiModel`
```

here and everywhere

Reply:
Also, explicit mention "if you want to read more about it, please look into original class :class:`...`"